src/config/dynamic-theme-fixes.config

```diff
@@ -30885,3 +30885,17 @@ CSS
 .zyimage {
     background-color: var(--darkreader-neutral-text) !important;
 }
+
+================================
+
+pir2.forumeiros.com
+
+INVERT
+img[src*="latex"]
+
+================================
+
+app.estuda.com
```

Comment:
Please preserve the alphabetical order.

Reply:
Hi Alexander!

Sorry for not inserting my fixes in alphabetical order, i should have read the documentation first.